Build/Customizations/DatabaseTestSetup.cs

```diff
@@ -101,15 +101,9 @@ public void ExecuteTests (TestExecutionContext context, Action<TestExecutionCont
     appConfig.SetAppSetting("Password", password);
 
     appConfig.WriteToFile(configFile);
+    File.Copy(configFile, configFile.Parent / "testhost.dll.config", true);
+    File.Copy(configFile, configFile.Parent / "testhost.x86.dll.config", true);
```

Comment:
@a-ctor copying the config for `dotnet test` here and in WebTestingTestSetup.cs.